CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.5)
 project(foonathan_memory_vendor VERSION "0.5.0")
 
 find_package(foonathan_memory QUIET)
+find_package(ament_cmake_copyright QUIET)
+find_package(ament_cmake_lint_cmake QUIET)
+find_package(ament_cmake_xmllint QUIET)
```

Comment:
@Blast545 consider moving these into the `if (BUILD_TESTING) ... endif()` clause.